pkg/internal/example/example.go

```diff
@@ -0,0 +1,34 @@
+// Package example contains helpers for implementing testable examples.
```

Comment:
Do you have an example in mind? I'm trying to understand how this would be consumed.

Reply:
Oh right I had set them up to be error-less, but it disrupts the test suite, so I switched back to early returns instead.